### PR TITLE
Mi 568 squads mobile default

### PR DESCRIPTION
**Pull request by nensidosari**
## Changes
- go to /squads/discover/my when user has squads on mobile/tablet
- go to /squads/discover when user doesn't have squads on mobile/tablet

MI-568 #done

### Preview domain
https://mi-568-squads-mobile-default.preview.app.daily.dev

### CODE REVIEW

File: packages/webapp/components/FooterNavBar.tsx [hunk 1]
```diff
@@ -37,9 +65,48 @@ export default function FooterNavBar({
...
+  const { user, squads } = useContext(AuthContext);
   const feedName = getFeedName(router.pathname, { hasUser: !!user });
   const activeNav = useActiveNav(feedName);
+
+  const hasSquads = squads?.length > 0;
+  const squadsUrl = hasSquads
+    ? squadCategoriesPaths['My Squads']
+    : squadCategoriesPaths.discover;
+
+  const tabs: (FooterTab | ReactNode)[] = [
```
Comment (nensidosari) [inline packages/webapp/components/FooterNavBar.tsx:77]: moved from FooterNavBarTabs component

File: packages/shared/src/components/sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -86,6 +86,10 @@ export default function Sidebar({
     hasUser: !!user,
     hasFiltered: !alerts?.filter,
   });
+  const hasSquads = squads?.length > 0;
+  const squadsUrl = hasSquads
+    ? `${webappUrl}squads/discover/my`
```
Comment (sshanzel) [inline packages/shared/src/components/sidebar/Sidebar.tsx:91]: We can probably reuse the paths from `squadCategoriesPath`.

Comment (nensidosari) [reply packages/shared/src/components/sidebar/Sidebar.tsx]: updated

File: packages/webapp/components/FooterNavBar.tsx [hunk 3]
```diff
@@ -15,14 +16,41 @@ import useActiveNav, {
...
-import { FooterNavBarTabs, tabs } from './footer/FooterNavBarTabs';
+import { FooterNavBarTabs } from './footer/FooterNavBarTabs';
+import { FooterPlusButton } from './footer/FooterPlusButton';
 
 interface FooterNavBarProps {
   showNav?: boolean;
   post?: Post;
 }
 
+const Notifications = ({ active }: { active: boolean }): JSX.Element => {
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/webapp/components/FooterNavBar.tsx:39]: Why did we move it though?

Comment (rebelchris) [inline packages/webapp/components/FooterNavBar.tsx:39]:
```suggestion
const Notifications = ({ active }: { active: boolean }): ReactElement => {
```

File: packages/webapp/components/FooterNavBar.tsx [hunk 4]
```diff
@@ -37,9 +65,51 @@ export default function FooterNavBar({
...
 }: FooterNavBarProps): ReactElement {
   const router = useRouter();
-  const { user } = useContext(AuthContext);
+  const { user, squads } = useContext(AuthContext);
   const feedName = getFeedName(router.pathname, { hasUser: !!user });
   const activeNav = useActiveNav(feedName);
+
+  const hasSquads = squads?.length > 0;
+  const squadsUrl = hasSquads
+    ? squadCategoriesPaths['My Squads']
```
Comment (rebelchris) [inline packages/webapp/components/FooterNavBar.tsx:74]: Got so confused by this string 😂
Not sure why we didn't make it camelCase or something like that.

**Review by rebelchris — APPROVED** [review]
Nothing blocking

**Pull request merged**